text_based_scenarios/_0_2_5_add_mre_json.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+
+def add_mre_json(mongo_db):
+    text_based_config = mongo_db['textBasedConfig']
+    
+    current_script_dir = os.path.dirname(os.path.abspath(__file__))
+    json_folder = os.path.join(current_script_dir, 'mre-json-files')
+    
+    # Get all JSON files in the folder
+    json_files = [f for f in os.listdir(json_folder) if f.endswith('.json')]
+    
+    for file in json_files:
+        file_path = os.path.join(json_folder, file)
+        with open(file_path, 'r') as f:
```

Comment:
The python `open` call always needs an extra parameter for me, `encoding='utf-8'`.  I was going to say that Windows needs it, but apparently you don't need it...